### DISCUSSION TITLE
How to Start Github OpenSource Journey

### DISCUSSION THREAD

**Discussion by neon-script** [community/community#106793] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Select one topic which interest you
create issues related to that topic
Find bugs and report them
Contribute in that project

**Comment by gil-son** [comment]
Hello,

this [video](https://www.youtube.com/watch?v=eNDuqcrLTLs) can support you

**Comment by neon-script** [comment]
But I create this discussion on how to contribute on open source project.
Ps- I already Know but i want those doesn't have any knowledge can create here issues.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @YOGINIMAHIMA1! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by neon-script** [comment]
Thanks for your suggestion **@ghostinhershell**
